dask/dataframe/core.py

```diff
@@ -2477,6 +2477,11 @@ def memory_usage(self, index=True, deep=False):
         result = self.map_partitions(M.memory_usage, index=index, deep=deep)
         return delayed(sum)(result.to_delayed())
 
+    def __divmod__(self, other):
+        res1 = self // other
+        res2 = self % other
+        return res1, res2
```

Comment:
Would be good to add `__rdivmod__` here as well 🙂

Reply:
Done